src/controller/jobmonitor/monitor.go

```diff
@@ -143,7 +143,7 @@ func (w *monitorController) StopRunningJobs(ctx context.Context, jobID string) e
 }
 
 func (w *monitorController) stopJob(ctx context.Context, jobID string) error {
-	tasks, err := w.taskManager.List(ctx, &q.Query{Keywords: q.KeyWords{"job_id": jobID}})
+	tasks, err := w.taskManager.List(ctx, &q.Query{Keywords: q.KeyWords{"JobID": jobID}})
```

Comment:
why do we change `job_id` to `JobID`? could you please elaborate it in details

Reply:
https://github.com/goharbor/harbor/blob/v2.2.0/src/controller/blob/controller.go#L233
Both job_id and JobID can work,  but I prefer to use JobID

Reply:
https://github.com/goharbor/harbor/blob/62223bd36d2a07f2a0f1982013f639cbc1991bd9/src/lib/orm/query.go#L136
Both can work.

Reply:
As mentioned in this code: https://github.com/goharbor/harbor/blob/62223bd36d2a07f2a0f1982013f639cbc1991bd9/src/lib/orm/query.go#L136
`FooBar` will be convert to snake_case of `foo_bar`.